spec/unit/hodor/environment_spec.rb

```diff
@@ -15,8 +15,11 @@ module Hodor
       # Public methods
       it { should include :erb_sub }
       it { should include :erb_load }
+      it { should include :yml_expand}
```

Comment:
YmlTools is included in environment so test succeeds.  I suspect this module may disappear entirely and be absorbed into yml_source once the configs are all loaded using the new approach.